src/AppInstallerCommonCore/Registry.cpp

```diff
@@ -422,6 +422,21 @@ namespace AppInstaller::Registry
         }
     }
 
+    void Key::SetValue(const std::wstring& name, const std::wstring& value, DWORD type) const

```

Comment:
As a user, it would be really nice to know the key and value that is being set if verbose logging is enabled